### PR TITLE
Adding support for long custom attributes

### PR DESCRIPTION
**Pull request by JeffAshton**
The UI supports long integers, just the client doesn't.

image

### CODE REVIEW

File: src/LaunchDarkly.Client/Operator.cs [hunk 1]
```diff
@@ -158,9 +123,26 @@ public static bool Apply(string op, JValue uValue, JValue cValue)
             return false;
         }
 
+        private static bool TryCompareNumericValues(JValue x, JValue y, out int result)
+        {
+            if (!IsNumericValue(x) || !IsNumericValue(y))
```
Comment (JeffAshton) [inline src/LaunchDarkly.Client/Operator.cs]: Just like before, only numeric values are compared

File: src/LaunchDarkly.Client/Operator.cs [hunk 2]
```diff
@@ -158,9 +123,26 @@ public static bool Apply(string op, JValue uValue, JValue cValue)
...
+            {
+                result = default(int);
+                return false;
+            }
+
+            result = x.CompareTo(y);
```
Comment (JeffAshton) [inline src/LaunchDarkly.Client/Operator.cs]: Comparison logic between different numeric value types is already implemented by Newtonsoft:

https://github.com/JamesNK/Newtonsoft.Json/blob/master/Src/Newtonsoft.Json/Linq/JValue.cs#L244

File: test/LaunchDarkly.Tests/OperatorTest.cs [hunk 3]
```diff
@@ -176,6 +176,11 @@ public void Apply_Contains_SupportedTypes_ReturnsFalse(object userValue, object
         [Theory]
         [InlineData(10, 11)]
```
Comment (JeffAshton) [inline test/LaunchDarkly.Tests/OperatorTest.cs]: All of these additional tests do the same thing.  Edge cases with combinations of different value types

**Comment by eli-darkly** [comment]
I'm retargeting this to a branch so I can resolve some conflicts with the current release, then we'll merge it. Thanks for the PR.

**Comment by JeffAshton** [comment]
Thanks @eli-darkly
